cmd/launcher/updater.go

```diff
@@ -63,25 +64,34 @@ func createUpdater(
 	var stop func()
 	stopChan := make(chan bool)
 
+	binaryName := path.Base(binaryPath)
+
 	return &actor.Actor{
 		Execute: func() error {
```

Comment:
Are we missing an error check for this `Execute` call? 

```
if err != nil {
    log.Fatal(err)
}
```

Reply:
That should be handled by the caller.

Reply:
Gotcha!